File: problem005/problem005.cpp
#include <iostream>

bool is_multiple(unsigned long long n)
{
	bool b = true;
	for (unsigned long long i = 2; i <= 20; ++i)
	{
		if (n % i)
		{
			b = false;
			break;
		}
	}
	return b;
}

int main()
{
	unsigned long long n = 20;
	while (!is_multiple(n))
		++n;

	std::cout << n << std::endl;
	return 0;
}

File: problem005/problem005.py
#!/usr/bin python

def is_multiple( n ):
	b = True
	for i in range(2,21):
		if (n%i != 0):
			b = False;
			break

	return b;


n = 20
while ( not is_multiple(n) ):
	n += 1;
else:
	print n
	
